flink-streaming-java/src/test/java/org/apache/flink/streaming/api/environment/RemoteStreamExecutionEnvironmentTest.java

```diff
@@ -57,9 +53,6 @@
 /**
  * Tests for the {@link RemoteStreamEnvironment}.
  */
-@RunWith(PowerMockRunner.class)
```

Comment:
If so, can we remove power mockito all from this test?

Reply:
Done!